modules/cnf-programming-the-sriov-fec-operator-acc100.adoc

```diff
@@ -0,0 +1,345 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="configuring-the-sr-iov-fec-operator-intel-vran-dedicated-accelerator-acc100_{context}"]
+= Configuring the SR-IOV-FEC Operator for the Intel® vRAN Dedicated Accelerator ACC100
+
+This section describes how to program the Intel® vRAN Dedicated Accelerator ACC100.
```

Comment:
Please replace this sentence with information that states the benefit of programming the card.